vita-latina.csl

```diff
@@ -0,0 +1,305 @@
+        <group delimiter=", ">
+          <text variable="collection-title" form="short"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="numero-collection">
+    <choose>
+      <if type="book chapter thesis report paper-conference" match="any">
+        <choose>
+          <if variable="collection-number">
+            <text variable="collection-number"/>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publication">
+    <choose>
+      <if type="thesis">
+        <group prefix=", ">
+          <text value="Thèse de doctorat soutenue à "/>
+          <text macro="lieu-edition"/>
+          <text macro="date-publication" prefix=", "/>
+          <text value="sous la direction de [DIRECTEUR(S) DE THESE]"/>
+        </group>
+      </if>
+      <else>
+        <group prefix=", " delimiter=", ">
+          <text macro="lieu-edition"/>
+          <text macro="editeur"/>
+          <text macro="titre-collection" quotes="true"/>
+        </group>
+        <text macro="numero-collection" prefix=" "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages-chapitre">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text variable="volume" suffix=" :"/>
+        <text variable="page" prefix=", p. "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="pages-article">
+    <choose>
+      <if type="article-journal">
+        <text variable="page" prefix=", p. "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="type-acces">
+    <group delimiter=" - ">
+      <text variable="URL" text-decoration="underline" prefix=" [" suffix="]"/>
+      <choose>
+        <if variable="URL">
+          <group prefix=" consulté le ">
+            <date variable="accessed">
+              <date-part name="day" suffix="/"/>
+              <date-part name="month" form="numeric" suffix="/"/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <!-- INSTRUCTIONS DE FORMATAGE DES CITATIONS -->
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true" disambiguate-add-year-suffix="true">
+    <layout suffix="." delimiter=" ; ">
+      <group delimiter=" : ">
+        <group delimiter=" ">
+          <text macro="auteurs-court"/>
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+        </group>
+        <text macro="indication-localisation"/>
+      </group>
+    </layout>
+  </citation>
+  <!-- INSTRUCTIONS DE FORMATAGE DE LA BIBLIOGRAPHIE -->
+  <bibliography et-al-min="11" et-al-use-first="7" subsequent-author-substitute="&#8212;" entry-spacing="0">
+    <sort>
+      <key macro="contributeurs-tri"/>
+      <key variable="issued"/>
+    </sort>
+    <layout suffix=".">
+      <group delimiter=" ">
+        <text macro="auteurs" prefix="  "/>
```

Comment:
(Those are `&#8195;` em spaces, by the way, which the formatter currently doesn't escape; http://www.codetable.net/decimal/8195)